cmd/proxy/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+
+	flag "github.com/spf13/pflag"
+	"google.golang.org/grpc"
+
+	pb "github.com/improbable-eng/etcd-cluster-operator/api/proxy"
+)
+
+type proxyServer struct {
+	pb.UnimplementedProxyServer
+}
+
+func main() {
+	// Setup defaults for expected configuration keys
+	var apiPort = flag.Int("api-port", 8080, "Port to serve the API on.")
```

Comment:
👍 this is fine for now - I'd definitely recommend an `options` struct with an `AddFlags(*pflag.FlagSet)`  method which uses `fs.IntVar(&opts.APIPort, ...)`in future though.